examples/websocket-basic-sample/websocket_basic_sample.bal

```diff
@@ -27,13 +27,20 @@ service<http:WebSocketService> basic bind { port: 9090 } {
 
         if (text == "ping") {
             io:println("Pinging...");
-            caller->ping(pingData) but { error e => log:printError("Error sending ping", err = e) };
+            caller->ping(pingData) but {
+                error e => log:printError("Error sending ping", err = e)
+            };
         } else if (text == "closeMe") {
-            caller->close(1001, "You asked me to close the connection")
-            but { error e => log:printError("Error occurred when closing the connection", err = e) };
+            caller->close(1001, "You asked me to close the connection") but {
+                error e => log:printError(
+                               "Error occurred when closing the connection",
+                               err = e)
```

Comment:
Can't we bring these lines up?

Reply:
All BBE should only contain 80 characters in a given line. Since these are longer than 80 characters, examples are not shown in the expected way in BBE.
Please take a look at https://ballerina.io/learn/by-example/websocket-basic-sample.html